src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -168,10 +174,57 @@ private void handleExit() {
         primaryStage.hide();
     }
 
+    @FXML
+    private void displayParticipantList() {
+        listPanel = new EntityListPanel(logic.getFilteredParticipantList());
+
+        entityListPanelPlaceholder.getChildren().set(0, listPanel.getRoot());
+        entityListPanelPlaceholder.setStyle("-fx-background-color: #5d6d7e");
+        logger.info("Color of entity list placeholder is: " + entityListPanelPlaceholder.getStyle());
+    }
+
+    @FXML
+    private void displayTeamList() {
+        listPanel = new EntityListPanel(logic.getFilteredTeamList());
+        entityListPanelPlaceholder.getChildren().set(0, listPanel.getRoot());
+        entityListPanelPlaceholder.setStyle("-fx-background-color:#abb2b9");
+        logger.info("Color of entity list placeholder is: " + entityListPanelPlaceholder.getStyle());
+
+    }
+
+    @FXML
+    private void displayMentorList() {
+        listPanel = new EntityListPanel(logic.getFilteredMentorList());
+        entityListPanelPlaceholder.getChildren().set(0, listPanel.getRoot());
+        entityListPanelPlaceholder.setStyle("-fx-background-color: #17202a");
+        logger.info("Color of entity list placeholder is: " + entityListPanelPlaceholder.getStyle());
+
+    }
+
     public EntityListPanel getListPanel() {
         return listPanel;
     }
 
+    /**
+     * Disarms or resets all buttons so that a new command can be carried out.
+     * The new command will arm a new button.
+     */
+    private void disarmAllButton() {
+        //TODO: shorten this
+        //Any ideas on how to shorten this method?
```

Comment:
An array of buttons? idk tbh

Reply:
Think he means store all the references to buttons in an array so you can just filter and map over to disarm

Reply:
haha yea like a Button[] of all the buttons